deploy/runtime/defaults/Generic Battery_Residential.json

```diff
@@ -71648,7 +71648,7 @@
     "ui_copy_batt_discharge_percent_5": 25.0,
     "ui_copy_batt_discharge_percent_6": 25.0,
     "ui_net_capital_cost": 46901.9,
-    "ui_pv_capacity": 4.0,
+    "ui_plant_capacity": 3.335046e-317,
```

Comment:
It looks like this is potentially the cause of the Travis failure:

![image](https://user-images.githubusercontent.com/5530592/197071084-b8a8257c-7ca9-4044-be1a-25963cb87ac2.png)

Maybe reverting the value to the original value will fix the issue? That or a form is out of date.